src/vue/components/panel-compact.vue

```diff
@@ -46,28 +46,33 @@
           </div>-->
         </div>
       </div>
+
       <div
           class="alpheios-panel__tab-panel alpheios-panel__content_no_top_padding alpheios-panel__tab-panel--fw alpheios-panel__tab__definitions"
-          v-if="$store.getters['ui/isActiveTab']('definitions')"
+          v-show="$store.getters['ui/isActiveTab']('definitions')"
           data-alpheios-ignore="all"
-          >
+      >
         <div class="alpheios-lookup__panel">
           <lookup
               :name-base="`panel-defs`"
           />
         </div>
-        <div v-if="$store.getters['app/defDataReady']">
-          <div :key="definition.ID"
-               class="alpheios-panel__contentitem" v-for="definition in formattedShortDefinitions">
-            <shortdef :definition="definition" :languageCode="$store.state.app.languageCode"></shortdef>
-          </div>
-          <div class="alpheios-panel__contentitem alpheios-panel__contentitem-full-definitions"
-               v-html="formattedFullDefinitions"></div>
-        </div>
-        <div v-else>
-          {{ l10n.getText('PLACEHOLDER_DEFINITIONS') }}
```

Comment:
so with this change we won't see anything that indicates definitions are loading until we have some?

Reply:
My reasons were that if defs are not available (which is the state during the process of lexical query running), nobody will be able to see this message: panel is closed automatically when new request starts, and with "definitions" button disabled until defs data is available, nobody will be able to get to to this panel and see the message. What do you think? Are there any scenarios when this message will be visible to the user?

Reply:
I guess it depends upon what we decide to do for #389

Reply:
Since we might need to do changes there for #389, I decided to put it back until then: it's always easier to remove the code than to write a now one.